contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: What is the difference between web analytics and product analytics?
+sidebar: Docs
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+- [Web analytics](/web-analytics) is for tracking website metrics. It's for marketing websites, blogs, or high volume B2C apps where users aren't logged in.
+- [Product analytics](/product-analytics) is for tracking your product. It's more powerful than web analytics. It's for web and mobile apps.
+
+<ComparisonTable column1="Web analytics" column2="Product analytics">
+  <ComparisonRow column1={"One, pre-built"} column2={"Unlimited, customizable"} feature="Dashboards" />
+  <ComparisonRow column1={false} column2={true} feature="Custom insights" />
+  <ComparisonRow column1={"Pageview, pageleave, conversions"} column2={"Any, custom"} feature="Events" />
+  <ComparisonRow column1={"Limited, pre-defined"} column2={"Any, custom"} feature="Properties" />
+  <ComparisonRow column1={"Small teams, marketers"} column2={"Engineering, product, data teams"} feature="Built for" />
+  <ComparisonRow column1={"At-a-glance view of website traffic"} column2={"Analysis of product usage and user behavior"} feature="Use case" />
+</ComparisonTable>
+
+## When should you use web analytics?
+
+Web analytics gives you a single, pre-built [dashboard](https://us.posthog.com/web) for a high-level overview of your site. It's about collecting data in the aggregate, as opposed to tracking specific interactions from specific users.
+
+It can help you figure out which pages and products are most popular and resonate the most with people.
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_light_mode_2024_10_be53cf5325.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/web_analytics_top_dark_mode_2024_10_6aa6dc9aeb.png"
+  alt="Web analytics dashboard"
+  classes="rounded"
+/>
+
+You can use web analytics to find out:
+
+<details>
+  <summary>How many <strong>visitors</strong> were there in recent days, weeks, and months?</summary>
```

Comment:
```suggestion
<details>
  <summary>How many visitors were there in recent days, weeks, and months?</summary>
```

Love these dropdowns, very cool, but you don't need to bold text. It doesn't render well on preview.

![Screenshot 2024-11-04 at 17 54 48](https://github.com/user-attachments/assets/734f6db5-a3e7-46c8-8de1-304d0db95c3e)